cmd/cli/app/quickstart/quickstart.go

```diff
@@ -137,9 +153,12 @@ var cmd = &cobra.Command{
 // quickstartCommand is the quickstart command
 //
 //nolint:gocyclo
-func quickstartCommand(_ context.Context, cmd *cobra.Command, conn *grpc.ClientConn) error {
+func quickstartCommand(_ context.Context, cmd *cobra.Command, inConn *grpc.ClientConn) error {
+	var err error
+	var conn *grpc.ClientConn
+	conn = inConn
```

Comment:
What's the purpose of copying the connection here? Can't we just reuse the variable?

Reply:
We can, yes 👍 I have no other reason other than I thought it'd better to be more explicit to say we are creating a new connection, but I'm happy to change this 👍 

As for the safety of not leaving a connection open, my understanding is we should be okay even if we overwrite the incoming variable as long as we do a `defer conn.Close()` on both and we do that right after the create part.

Reply:
Fixed it in https://github.com/stacklok/minder/pull/2458/commits/d1d74f8926b95df44986c06f5db9d315f9e72cc3 👍